runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -133,16 +138,15 @@ var saved_K_map: dict<any>
 var saved_plus_map: dict<any>
 var saved_minus_map: dict<any>
 
-
 def InitScriptVariables()
   if exists('g:termdebug_config') && has_key(g:termdebug_config, 'use_prompt')
-    way = g:termdebug_config['use_prompt'] ? 'prompt' : 'terminal'
+    way = g:termdebug_config['use_prompt'] ? Way.Prompt : Way.Terminal
   elseif exists('g:termdebug_use_prompt')
-    way = g:termdebug_use_prompt
```

Comment:
The original version seems buggy. `way` is expected to be 'prompt' or 'terminal', while `g:termdebug_use_prompt` is `1` or `0` according to `:h termdebug_use_prompt`

Reply:
I also got confused when I saw it. But it actually works.